mitochondria-pipeline.inputs.json

```diff
@@ -1,6 +1,6 @@
 {
   "##_COMMENT1": "Inputs",
-  "MitochondriaPipeline.wgs_aligned_input_bam": "gs://gatk-best-practices/mitochondria-pipeline/G97753.NA12878.bam",
+  "MitochondriaPipeline.wgs_aligned_input_bam_or_cram": "gs://gatk-best-practices/mitochondria-pipeline/G97753.NA12878.bam",
```

Comment:
Is this a change I should propagate to the gatk repo?

Reply:
It isn't required but would be very helpful so that the next time the wdl is updated in this repo we don't forget to always change the input variable name.

Reply:
Ok, I'll make a PR in gatk (better to keep things easier for next time if possible).